src/reactions/pin.ts

```diff
@@ -0,0 +1,33 @@
+import { ReactionDef } from './reaction-def';
+import { MessageEmbed } from 'discord.js';
+
+export const pin: ReactionDef = {
+  emoji: '📌',
+  description: 'Sends message to the current user as a DM',
+  command: async (reaction) => {
+    try {
+      const users = await reaction.users.fetch();
+      // Get user that reacted with the pushpin emoji
+      const user = users.first();
+      const pinnedEmbed = new MessageEmbed()
+        .setColor('#0099ff')
+        .setTitle("Here's your pinned message buddy!")
```

Comment:
At some point I'd like to change this verbiage to something more professional...

Reply:
Got any ideas? We can change i now so we don't forget

Reply:
Something like "Pinned Message:" would be sufficient, IMO.